thirdparty/hairer/radau5_c_py.pxd

```diff
@@ -15,31 +15,47 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-from numpy cimport int64_t
+from numpy cimport int32_t

```

Comment:
I'm a bit worried about using int32 here. We will quickly run into overflows. Can we not use something else?

Reply:
Keeping this as a future improvment.